README.md

```diff
@@ -1,10 +1,10 @@
 # 2D-plotter
 To create a 2D plotting machine which can generate a binary image on a paper using a stylus.
 
-Objectives
+## Objectives
 
-> extract coordinates of pixels from a binary image
-> map out the points to real-life-size paper dimensions (in cm)
-> insert the points into a readable file format for the arduino
-> create a arduino program to control the motors to move the stylus
-> create a 3D CAD file of the 2D Plotter machine
+- [ ]  extract coordinates of pixels from a binary image
```

Comment:
it makes the objectives behave like checkboxes. Just put an x inside the space when the task is done.

Reply:
Thanks brother